metaflow/plugins/airflow/airflow_utils.py

```diff
@@ -424,7 +424,17 @@ def _get_sensor(name):
             )
         return S3KeySensor
     elif name == SensorNames.SQL_SENSOR:
-        from airflow.sensors.sql import SqlSensor
+        try:
+            from airflow.sensors.sql import SqlSensor
+        except ImportError as e:
+            try:
+                from airflow.providers.common.sql.sensors.sql import SqlSensor
+            except ImportError as e:
+                raise AirflowSensorNotFound(
+                    "This DAG requires a `SqlSensor`. "
+                    "Install the Airflow SQL provider using : "
+                    "`pip install apache-airflow-providers-common-sql`"
+                )
```

Comment:
Had to make this change so that SQL sensors can be supported. Airflow Recently [removed SQL sensors](https://airflow.apache.org/docs/apache-airflow/stable/release_notes.html#dbapihook-and-sqlsensor-have-moved-24836) from Airflow v2.4.0 . This is why we have this additional code path added here.